src/main/scala/nl/biopet/utils/tool/ToolCommand.scala

```diff
@@ -39,4 +36,12 @@ trait ToolCommand[Args] extends Logging {
       .parse(args, emptyArgs)
       .getOrElse(throw new IllegalArgumentException)
   }
+
+  def usageText: String = {
+    // Prepend code block with four spaces in concordance with Markdown specification.
+    val usage = new StringBuffer()
+    val usageLines: Array[String] = argsParser.usage.split("\n")
```

Comment:
I think it might be better to parse the arguments itself instead of the usage text